tests/js/app-tests.js

```diff
@@ -158,15 +156,18 @@ module.exports = {
             }
         };
         Realm.deleteFile(realmConfig);
-
+        console.log("KAT 1");
```

Comment:
debugging leftover ?

btw, in bulgarian this means "Traffic Police 1" 😀

Reply:
Indeed so - thanks.

Reply:
Clean up a bit :-)